x/concentrated-liquidity/README.md

```diff
@@ -234,12 +234,18 @@ One draw back of this implementation is the requirement to create many ticks tha
 
 - **Request**
 
-This message allows LPs to provide liquidity between `LowerTick` and `UpperTick` in a given `PoolId.
+This message allows LPs to provide liquidity between `LowerTick` and `UpperTick` in a given `PoolId`.
 The user provides the amount of each token desired. Since LPs are only allowed to provide
 liquidity proportional to the existing reserves, the actual amount of tokens used might differ from requested.
 As a result, LPs may also provide the minimum amount of each token to be used so that the system fails
 to create position if the desired amounts cannot be satisfied.
 
+Three KV stores are initialized when a position is created:
+
+1. `Position ID -> Position` - This is a mapping from a unique position ID to a position object. The position ID is a monotonically increasing integer that is incremented every time a new position is created.
+2. `Owner | Pool ID | Position ID -> Position ID` - This is a mapping from a composite key of the owner address, pool ID, and position ID to the position ID. This is used to keep track of all positions owned by a given owner in a given pool.
+3. `Pool ID -> Position ID` - This is a mapping from a pool ID to a position ID. This is used to keep track of all positions in a given pool.
```

Comment:
Does it make sense to have `Pool ID | Owner -> Position ID` so that we have another level of granularity for queries?

Reply:
Let's also address this in follow-up issue please

Reply:
The information you could extract from `Pool ID | Owner` is extractable via `Owner | Pool ID`

Reply:
True, agreed